python/servo/try_parser.py

```diff
@@ -104,7 +107,11 @@ def update_name(self):
 def handle_preset(s: str) -> Optional[JobConfig]:
     s = s.lower()
 
-    if any(word in s for word in ["linux"]):
+    if s == "linux-arm":
+        return JobConfig("Linux ARM", Workflow.LINUX_ARM)
+    elif any(word in s for word in ["arm"]):
+        return JobConfig("Linux ARM", Workflow.LINUX_ARM)
```

Comment:
Let's join these two:
```suggestion
    if s == "linux-arm" or any(word in s for word in ["arm"]):
        return JobConfig("Linux ARM", Workflow.LINUX_ARM)
```